packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.ClientModel/test/Providers/Definitions/TestData/ModelReaderWriterContextDefinitionTests/ValidateObsoleteFrameworkTypeHasAttributeSuppression.cs

```diff
@@ -8,9 +8,9 @@
 
 namespace Sample
 {
-#pragma warning disable CS0618 // global::Microsoft.TypeSpec.Generator.ClientModel.Tests.Providers.Definitions.ModelReaderWriterContextDefinitionTests.ObsoleteFrameworkType is obsolete and may be removed in future versions.
+#pragma warning disable CS0618 // Microsoft.TypeSpec.Generator.ClientModel.Tests.Providers.Definitions.ModelReaderWriterContextDefinitionTests+ObsoleteFrameworkType is obsolete and may be removed in future versions.
     [global::System.ClientModel.Primitives.ModelReaderWriterBuildableAttribute(typeof(global::Microsoft.TypeSpec.Generator.ClientModel.Tests.Providers.Definitions.ModelReaderWriterContextDefinitionTests.ObsoleteFrameworkType))]
-#pragma warning restore CS0618 // global::Microsoft.TypeSpec.Generator.ClientModel.Tests.Providers.Definitions.ModelReaderWriterContextDefinitionTests.ObsoleteFrameworkType is obsolete and may be removed in future versions.
+#pragma warning restore CS0618 // Microsoft.TypeSpec.Generator.ClientModel.Tests.Providers.Definitions.ModelReaderWriterContextDefinitionTests+ObsoleteFrameworkType is obsolete and may be removed in future versions.
```

Comment:
Why did this change?

Reply:
Same as https://github.com/microsoft/typespec/pull/8313#discussion_r2312753891.